### ISSUE TITLE
Empty Toast When Attempting to Set Unverified Email as Primary

### ISSUE THREAD

**Issue by unrenamed** [unkeyed/unkey#2514]
### Preliminary Checks

- [X] I have reviewed the documentation: https://unkey.com/docs

- [X] I have searched for existing issues: https://github.com/unkeyed/unkey/issues

- [X] This issue is not a question, general help request, or anything other than a bug report directly related to Unkey. Please ask questions in our Discord community: https://unkey.com/discord.

### Issue Summary

When a user tries to make an unverified email address their primary email, the application shows an empty toast notification, providing no feedback or error message.

### Steps to Reproduce

1. Log into your account.
2. Navigate to the profile settings or account settings page.
3. Add a new email address but do not verify it.
4. Attempt to make the unverified email address your primary email.
5. Observe the toast notification.

### Expected behavior

When a user attempts to set an unverified email as their primary email:
- The toast notification should display an informative message such as, "Please verify your email before making it primary."
- The action should not be allowed, and the user should be instructed to verify their email first.

**Or the button "Make Primary" should not be available in an unverified email dropdown at all.**

### Screenshots

https://github.com/user-attachments/assets/cdc4747d-1f8c-4b58-aea8-ac34ccac9b69

### Version info

```markdown
- OS:
- Node:
- npm:
```

**Comment by SandipGyawali** [comment]
@unrenamed are you working on this issue ?

**Comment by unrenamed** [comment]
Let's wait for @chronark 's approval first. This issue may already be addressed as part of the auth/user account refactoring.. See https://github.com/unkeyed/unkey/issues/2201#issuecomment-2395019106

**Comment by chronark** [comment]
yeah, auth related things are currently being worked on by @mcstepp 
/award 50

**Comment by GARY121github** [comment]
@chronark Can I work on this issue?

**Comment by mcstepp** [comment]
no longer an issue after auth delivery
